dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -99,15 +99,11 @@ def setUpClass(cls):
         )
 
         cls.expected_properties = {
-            "best_fit_properties": [
-                8.646041719759628,
-                1.6999999999999997,
-                0.19403886939727638,
-            ],
-            "mean": [1.7085707836543698e16, 4.241852142820433, 1.0190038591415866],
-            "variance": [8.521811094505713e64, 305.76588081569196, 0.03984103474264823],
-            "skew": [4.987683961374356e48, 82.41830452500491, 0.5951548443693909],
-            "kurtosis": [7.262126433044066e129, 117436.2896499293, 0.6363254662738349],
+            "best_fit_attributes": [8.646041719759628],
+            "mean": 3315291431455125.5,
+            "variance": 3.2085541544027255e63,
+            "skew": 4.987683961374356e48,
+            "kurtosis": 7.262126433044066e129,
```

Comment:
Why do these values change for the unit test. If functionality is equivalent shouldnt these values stay the same?

Reply:
So we would need another fucntion to ensure the older code within the above if else statement is still functioning the same way.

Reply:
This is because the distribution was being calculated incorrectly in the original code.

Reply:
These are the values for the distribution, it is because the size of the distribution's scale parameter is so large.

Reply:
They are the moments from the distribution not from the actual data in the graph. They are about as large as before.